client/message/messages.js

```diff
@@ -68,21 +66,5 @@ Messages.propTypes = {
   messages: React.PropTypes.arrayOf(React.PropTypes.object).isRequired,
 }
 
-const byDateDescending = R.comparator((d1, d2) => d1.sent > d2.sent)
-
-const getGroupMessages = (state, {params}) => {
-  const groupUid = params.groupUid;
-  return {
-    messages: R.sort(byDateDescending, R.values(getMessagesByGroup(state, groupUid))),
-  }
-}
-
-const getStoryMessages = (state, {params}) => {
-  const storyUid = params.storyUid;
-  return {
-    messages: R.sort(byDateDescending, R.values(getMessagesByStory(state, storyUid))),
-  }
-}
-
-export const GroupMessages = connect(getGroupMessages)(Messages)
-export const StoryMessages = connect(getStoryMessages)(Messages)
+export const GroupMessages = connect(state=> ({messages: getGroupMessages})(Messages))
```

Comment:
The bracketing here is incorrect which was throwing some things off, and also we need to actually call these functions, here's what it looks like now:

```javascript
const stateToPropsGroup = (state, {params:{groupUid}}) => {
  return {
    messages: getGroupMessages(state, groupUid)
  }
}

const stateToPropsStory = (state, {params:{storyUid}}) => {
  return {
    messages: getStoryMessages(state, storyUid)
  }
}

export const GroupMessages = connect(stateToPropsGroup)(Messages)
export const StoryMessages = connect(stateToPropsStory)(Messages)

```